service/validator/validator.go

```diff
@@ -98,18 +99,18 @@ func validateServiceData(service *Service) error {
 }
 
 // isServiceKeysUnique checks uniqueness of service deps/tasks/events/params keys.
-func isServiceKeysUnique(service *Service) error {
+func isServiceKeysUnique(srv *service.Service) error {
```

Comment:
```suggestion
func isServiceKeysUnique(s *service.Service) error {
```
`s` is the name in the `Service`'s method (`(s *Service)`) so if you change it it will be good to  make it consistent accross the functions and methods.

Reply:
done